modules/authentication/src/Authentication.ts

```diff
@@ -17,6 +19,14 @@ import { TokenType } from './constants/TokenType';
 import { v4 as uuid } from 'uuid';
 import moment from 'moment';
 import { migrateLocalAuthConfig } from './migrations/localAuthConfig.migration';
+import {
```

Comment:
you're importing client types, and not only that you're importing from the dist folder. Remove this.